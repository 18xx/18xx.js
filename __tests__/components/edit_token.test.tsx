import * as React from 'react';
import * as renderer from 'react-test-renderer';

import EditToken from '../../src/components/edit_token';

describe('EditToken', () => {
  describe('#render', () => {
    const subject: React.ReactElement<EditToken> = (
      <EditToken onRemoveToken={jest.fn()} />
    );

    it('matches the rendered snapshot', () => {
      expect(renderer.create(subject)).toMatchSnapshot();
    });
  });
});
