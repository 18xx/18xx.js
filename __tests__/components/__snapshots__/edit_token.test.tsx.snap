// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditToken #render matches the rendered snapshot 1`] = `
<div
  id="tokenMenu"
>
  <button
    onClick={[Function]}
  >
    Remove Token
  </button>
</div>
`;
